browser/js/adminDash/song/adminDash.songCtrl.js

```diff
@@ -0,0 +1,40 @@
+app.controller('adminDashSongsCtrl', function ($scope, SongsFactory) {
+
+  SongsFactory.getSongs()
+    .then(function (songs) {
+      $scope.songs = songs;
+    });
+
```

Comment:
whoops, wasn't gulping! I forgot, this is needed to pull in all the songs when the controller boots up on the view switch. We actually didn't need the ng-click to call getSongs() when you click the 'Music' button so I deleted that.